.github/workflows/ci.yml

```diff
@@ -13,24 +13,56 @@ on:
 
 env:
   DOC_TROUBLESHOOTING: "Warnings like 'reference target not found' occur when Sphinx tries to interpret text as a Python object, but is unable to do so. This warning can often be fixed by surrounding text in double back ticks instead of single back ticks (e.g., by changing `y` to ``y``) so that it gets formatted as an in-line literal. For more information about addressing documentation build failures, please check out the documentation troubleshooting guide at: https://docs.plasmapy.org/en/latest/contributing/doc_guide.html#troubleshooting"
+  GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
 
 jobs:
 
 
   tests:
     name: ${{ matrix.name }}
     runs-on: ${{ matrix.os }}
+    timeout-minutes: 30
 
     strategy:
       fail-fast: false
 
       matrix:
         include:
 
+        - name: Tests, Python 3.12, skip slow tests, doctests, Ubuntu
+          os: ubuntu-latest
+          python: '3.12'
+          nox_session: tests-3.12(skipslow)
+
+        - name: Tests, Python 3.11, with code coverage, macOS
+          os: macos-latest
+          python: '3.11'
+          nox_session: tests-3.11(cov)
+
+        - name: Tests, Python 3.10, lowest direct dependencies, Windows
+          os: windows-latest
+          python: '3.10'
+          nox_session: tests-3.10(lowest-direct)
+
+        - name: Packaging, Python 3.12, Ubuntu
+          os: ubuntu-latest
+          python: '3.12'
+          nox_session: build
+
+        - name: Import PlasmaPy, Python 3.10, Windows
+          os: windows-latest
+          python: '3.10'
```

Comment:
Turns out that it is important to have the quotes around `3.10`.  Without them, GitHub interprets the string as $3.1$.  👀

Reply:
-_- how nice of GitHub 😆